tests/qir/test_bitreg_as_bit_8-QIRProfile.PYTKET.ll

```diff
@@ -0,0 +1,48 @@
+; ModuleID = 'test_bitreg_as_bit_8'
+source_filename = "test_bitreg_as_bit_8"
+
+%Result = type opaque
+%Qubit = type opaque
+
+@0 = internal constant [2 x i8] c"a\00"
```

Comment:
Is this expected?

Reply:
Yes, is it only this line you are worried about? Which point are you worried here?

Reply:
I was misreading it, thought `a` was being created as a register of size 2, but this is just the name, OK.